### DISCUSSION TITLE
zh/guide/drivers/189

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#28] [Announcements]
# zh/guide/drivers/189

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/189.html

**Comment by jinchunyou** [comment]
Failed init storage: ocr error:    天翼云盘重启服务器之后运行alist显示上面的错误，试过改密码，删除重新添加，重新启动服务，都没有效果。 。

**Reply by blycr** [reply]
同样的问题

**Reply by loomee** [reply]
因为登录时遇到了图形验证码，而没有给alist配置验证码识别服务就这样。
可以尝试先在浏览器登录云盘一些时间，让服务器认为是常用地点就没验证码。
还有是添加存储选择云盘客户端。
或者了解相关知识的话，给alist配置验证码识别服务也可以。

**Reply by SuperBill888** [reply]
添加类型：189CloudPC

**Comment by kevzcr** [comment]
无法保存信息  显示  Failed init storage: ocr error:

**Reply by kevzcr** [reply]
sorry,   应该添加天翼云盘客户端，我添加成天翼云盘了。

**Reply by SuperBill888** [reply]
添加类型：189CloudPC

**Comment by dvdrm** [comment]
V2没问题，V3保存不了

**Reply by yyddtj** [reply]
对

**Reply by LeiXuans** [reply]
现在似乎也有问题了

**Comment by miniidc** [comment]
无法使用根文件夹ID。输入还是挂载全盘。

**Comment by yyddtj** [comment]
failed get objs: failed to list objs: get page: status: 302 request url:天翼网盘总是出现这个问题，v3

**Reply by yyddtj** [reply]
哦哦我试试客户端

**Comment by fdwn** [comment]
错误
鉴权失败

**Comment by usami-muzugi** [comment]
我也是鉴权失败

**Comment by iqinglan** [comment]


**Reply by xwangiang** [reply]
最新版V3.4设置了加密文件夹也没有效果。而且最近用RaiDrive挂载到本地提示无取访问，表示不知道什么原因。

**Reply by cakiihana** [reply]
新版raidrive挂载后，能够看到云盘文件列表，但是无法复制到本地，或者粘贴到服务器，也无法直接右键新建文本，windows提示无权访问，不知道是谁的问题。。。

**Comment by BrandonStudio** [comment]
1. 用于登录的密码能不能设个掩码啊？存储的时候希望也做个安全存储，我现在看着我这个大明文心慌。如果不行的话能不能加个二维码登录或者cookies登录？
2. 我换了文件夹ID之后页面显示的还是根目录下的文件，不知道为什么

**Reply by BrandonStudio** [reply]
第2个问题我发现需要在网页上手动点一下右下角的刷新才可以
逻辑上应该在修改了配置之后就刷新一下

**Comment by HsinKwok** [comment]
后台做完存储设置，网页可以访问了，但是后面存储设置列表为空，看不到之前添加的记录。v3.4.0

**Comment by hnnyxycsy** [comment]
开秒传上传失败啊，是哪里的问题？

**Comment by ZJF-Kedaya** [comment]
有些视频线上看不了，但是用raidrive挂载到本地就可以看

**Comment by hongdou114514** [comment]
家庭空间怎么添加，只能添加个人空间吗？

**Reply by shiyest** [reply]
去v2版本中查看！

**Reply by FashionCrazy** [reply]
官网登录显示10TB，添加之后只有1TB
试了【天翼云盘】和【天翼云盘客户端】都一样。有老哥遇到了吗？

**Reply by hongdou114514** [reply]
> 去v2版本中查看！

什么意思，详细一点

**Reply by hongdou114514** [reply]
解决：https://alist.nn.ci/zh/guide/drivers/189.html

**Comment by xiaoTT** [comment]
Post "https://api.nn.ci/ocr/file/json": context deadline exceeded (Client.Timeout exceeded while awaiting headers  天翼云盘客户端莫名其妙就这样了  本来好好的

**Reply by 704103** [reply]
同样的问题，提示连接超时，夸克也这样，重新添加也不行，不知道是不是因为使用习惯问题，频繁登陆，我挂在手机上，用的时候才打开alist

**Reply by chengleigit** [reply]
一样一致添加不了天翼云盘

**Reply by vistatj** [reply]
遇到同样问题，天翼云盘不行，天翼云盘客户端也不行

**Reply by chengleigit** [reply]
有办法解决吗

**Comment by michealhansun** [comment]
need img validate code:
提示需要验证码？请问在哪里填入验证码

**Reply by vicacheung** [reply]
开下OCR

**Reply by michealhansun** [reply]
谢谢，可以了

**Comment by LeiXuans** [comment]
所有东西以及验证码啥的都配置好了，但就是下载速度很慢，这个怎么解决？？

**Comment by Chekhovara** [comment]
状态: 

添加完成后提示：设备ID不存在，需要二次设备校验

**Comment by gaoxiao228** [comment]
更新到3.11之后,天翼云盘 报错:failed get objs: failed to list objs: Post "https://api.nn.ci/ocr/file/json": x509: certificate signed by unknown authority

**Reply by ergwang** [reply]
同样的问题+1

**Reply by 13323688772** [reply]
同样问题+1

**Reply by ZJF-Kedaya** [reply]
+1

**Reply by hnnyxycsy** [reply]
+++1

**Reply by youtehub** [reply]
注意看文档，最后的  建议

**Reply by qqxpee** [reply]
这个随着部署alist挂载天翼云盘的人越来越多，使用的人也越来越多，如果是有自己的服务器的，最好还是自己部署一个。docker命令作者也提供了【docker run -d --restart=always -p 9898:9898 xhofe/ddddocr_server:main】

**Reply by youtehub** [reply]
好主意，后面使用 天翼网盘 自己在docker.挂载加载一个

**Comment by youtehub** [comment]
最新版   **v3.11.0**  中的解决办法是，将  **https**  改为  **http** , 修改的位置如图所示
建议打开这个选项，不然每次重新启动 **alist** 需要 **手动输入验证码**,或者更换成你自己部署的  ocr 验证码识别服务
![image](https://user-images.githubusercontent.com/18629921/219908528-6415016c-4e20-472e-a842-0722eb455a99.png)

**Reply by zjmazy** [reply]
强！终于解决了

**Reply by mtk2025** [reply]
厉害，感谢

**Comment by C-yd** [comment]
你需要权限才能执行此操作
你需要S-1-1-0提供的权限才能对此文件进行更改
这是什么问题？

**Comment by Winyer** [comment]
电脑打开https://h5.cloud.189.cn/main.html#/family这个页面显示的还是个人云没有家庭云，换了UA也不行，最后我是用手机浏览器抓包的家庭云ID😥

**Comment by Winyer** [comment]
unexpected EOF
复制时经常出现这个

**Comment by Winyer** [comment]
复制文件报错
updload fail,msg: <?xml version="1.0" encoding="UTF-8"?><Error><Code>BadDigest</Code><BucketName>media-shpd-fy-home</BucketName><RequestId>tx0000e28102-00643b9781</RequestId><HostId>text</HostId></Error>

**Comment by Winyer** [comment]
这个盘应该是有什么限制，webdav写入时只有几kb的速度，开启代理也是一样的

**Reply by Winyer** [reply]
破案了，webdav根本就无法写入，不是只有几kb的速度是无法写入！！！

**Comment by Winyer** [comment]
上传报错
Cannot read properties of undefined (reading 'status')

**Comment by wqhde** [comment]
使用RaiDrive在本地挂载Alist后，可以访问天翼云盘内容列表，但是没办法正常复制粘贴到本地。提示没有权限，请问这是什么情况。
我把天翼云盘和阿里云盘分别设置成不同的文件夹。同样的方法本地挂载后，阿里云盘内容可以正常读写。如果是通过网页访问就不存在问题。
使用环境：
威联通NAS（公网IPV6访问），安装Alist，

**Reply by Winyer** [reply]
一样的，不过我是复制不进去，移动云盘，阿里云，天翼云都这样，只有onedrive可以

**Reply by sumdream** [reply]
我用群晖装载挂载webdav也不行,同样提示是没有权限,能显示列表,但是没法复制东西.在alist主页里面点击下载速度很快.两种驱动都试了(天翼云和天翼云客户端)

**Comment by kevinzwq** [comment]
天翼云盘已开黄金会员，正常用天翼云盘客户端上传文件都无大小限制，但是用这个alist中天翼云盘客户端中上传大文件却出现了invalid part Size，但是如果点开秒传开关，即使是超大文件（前提是其他人已经上传同文件）同样可上传成功。针对这个alist中天翼云盘客户端中上传大文件却出现了invalid part Size这个该怎么解决？

**Comment by Winyer** [comment]
关闭ocr后怎么手动输入验证码？最近ocr识别验证码一直失效

**Comment by mtk2025** [comment]
Post "http://api.nn.ci/ocr/file/json": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
请问这是什么问题啊？

**Reply by mcok** [reply]
相同问题

**Reply by mtk2025** [reply]
ocr验证的问题，我手动打码了

**Comment by goooxyz** [comment]
自己搭了一个OCR接口,放出来给大家白嫖https://ddddocr.113111.xyz/ocr/file/json

2023/07/09<本接口为爱发电,希望有些大佬高抬贵手,不要将接口用于其他项目,最近经常持续高并发,每秒钟几十上百个请求,咱这小水管服务器求放过,目前已经设置了流量限制,如果已经在用的小伙伴出现速度慢或者请求失败最好自己搭一个>

同时建议大家有条件可以自己搭建OCR,作者本身有提供docker版本的ocr镜像,一键命令搭建,无需设置直接用;作者docker地址https://hub.docker.com/r/xhofe/ddddocr_server
搭建后默认使用方法 http://docker容器IP:端口/ocr/file/json;
如果你反向代理了域名以及配置了SSL那接口地址是 https://你的域名/ocr/file/json

**Reply by xsd116** [reply]
好人一生平安

**Reply by PieJEed** [reply]
谢谢哥

**Reply by youtehub** [reply]
必须赞一个，直接加载到docker 中就能正常运行，搭建后默认使用方法  http://docker容器IP:端口/ocr/file/json;

**Comment by heli9913** [comment]
failed get objs: failed to list objs: get page: <!DOCTYPE html><html><head><script type="text/javascript">var _uxt = _uxt || []; var _ux21st = (new Date()).getTime(); (function() { var ma = document.createElement('script'); ma.type = 'text/javascript'; ma.async = 'async'; ma.src = ('https:' == document.location.protocol ? 'https://ux.21cn.com' : 'http://ux.21cn.com') + '/api/htmlReportRest/getJs.js?pid=9A46A2CE574243A481E898A36E571DB2'; var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ma, s); })();</script><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"><meta name="renderer" content="webkit"><meta name="viewport" content="user-scalable=no,initial-scale=1,maximum-scale=1,minimum-scale=1,width=device-width,height=device-height"><meta name="apple-touch-fullscreen" content="yes"><meta name="apple-mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-status-bar-style" content="black"><title>天翼账号</title><link rel="shortcut icon" href="https://static.e.189.cn/open/login/images/logo_ico.png" type="image/png"><script>var login_mode = 1; var _ux_version = "V5.0"; var _ux21et = (new Date()).getTime();</script><link href="https://static.e.189.cn/open/login/page/web/v5.0/static/css/App.6a86e01c13096e3100729399365aee06.css" rel="stylesheet"></head><body><div class="web-login"><ul id="j-top-tab" class="top-tab clearfix"></ul><div id="j-logo" class="logo-wrap hide"><img width="100" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/web-logo.png" alt=""></div><div id="j-tab-list"><div id="j-qr-box" class="item qr-code-box hide"><div class="scan-inner pr"><div class="scan-box"><div class="bg-loading"><img id="j-qrcodeImage" src="" alt=""><div id="j-scan-status" class="sc-nwer"><div class="tip1">网络出错<br>请检查网络连接</div><div class="tip2"><p class="mb10">二维码已失效</p><p id="j-refresh" class="sm-btn sm-btn2">请点击刷新</p></div><div class="mask"></div></div></div><div class="tip">请关闭WiFi，在4G/5G状态下使用以下手机APP扫码</div><div id="j-support-list" class="support-list"><a title="小翼管家"><img class="ic-logo" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-itv.png"> <em>小翼管家</em> </a><a class="cp-link" target="_blank" rel="noopenner noreferrer" href="https://cloud.189.cn/web/static/download-client/index.html"> <img class="ic-logo" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-cloud.png"> <em>天翼云盘</em> <img class="dw-tip" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/load-tip.png"> </a><a title="微信"> <img class="ic-logo" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-wx.png"> <em>微信</em></a></div></div><div id="J-qrCodeLoginResult" class="scan-success hide"><p><img width="80" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/scan-success.png" alt=""></p><p class="sc-txt">扫码成功</p><p class="tip-txt">请按手机端提示操作</p><p id="j-qrcodeBack" class="sm-btn">重新扫码登录</p></div></div><div class="btm-inner"><a class="feedback fr" href="https://e.dlife.cn/help/registerHelp.do" target="_blank" rel="noopenner noreferrer">帮助</a> <span id="j-auto-group-qr" class="free-login hide"><i id="j-auto-login-qr" class="icon checkbox-icon1" data-index="0"></i> <span class="login-select"><em class="login-time-text">7天内自动登录</em> <img class="login-icon-arrow" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-arrow.png" alt=""><div class="login-option-list"><p class="option option-first active" data-time="1">7天内自动登录</p><p class="option" data-time="2">15天内自动登录</p><p class="option option-last" data-time="3">30天内自动登录</p></div></span><img class="login-tips" width="188" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/al-tip.png" alt=""></span></div></div><div id="j-psw-box" class="item psw-login-box login-box hide"><div id="j-accout-tip" class="error-tip hide2"><i class="icon error-tip-icon"></i> <span class="txt"></span></div><ul class="form-ul"><li><input type="text" id="userName" class="ui-input" autocomplete="off" placeholder="手机号 / 邮箱"> <span class="evet-box clear-btn" style="right: 18px;"><i class="icon clear-btn-icon"></i></span></li><li><input type="password" id="password" class="ui-input" placeholder="密码"> <span class="icon eye-icon1 pa-right"></span> <span class="evet-box clear-btn"><i class="icon clear-btn-icon"></i></span></li><li class="last"><input type="button" id="j-login" class="ui-button" value="登录"><div class="jump agreement-box clearfix"><span id="j-agreement-box" class="icon checkbox-icon1"><img class="xy-tip" width="92" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/xy-tip.png" alt=""></span><p class="agreement-txt">登录即同意<a target="_blank" rel="noopenner noreferrer" href="https://e.dlife.cn/_agreements.html">服务协议</a>和<a target="_blank" rel="noopenner noreferrer" href="https://e.dlife.cn/_agreements.html">隐私政策</a></p></div><div id="j-auto-group" class="free-login hide"><i id="j-auto-login" class="icon checkbox-icon1" data-index="0"></i> <span class="login-select"><em class="login-time-text">7天内自动登录</em> <img class="login-icon-arrow" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-arrow.png" alt=""><div class="login-option-list"><p class="option option-first active" data-time="1">7天内自动登录</p><p class="option" data-time="2">15天内自动登录</p><p class="option option-last" data-time="3">30天内自动登录</p></div></span><img class="login-tips" width="188" style="top: 19px;" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/al-tip2.png" alt=""></div></li></ul><div class="btm-inner clearfix"><span class="fr"><a id="j-forgot" target="_blank" rel="noopenner noreferrer" href="https://e.dlife.cn/getPassword.do">找回密码</a> <span>｜</span> <a id="j-e-about" target="_blank" rel="noopenner noreferrer" href="https://e.dlife.cn/register.do">注册</a> <span>｜</span> <a class="feedback" href="https://e.dlife.cn/help/registerHelp.do" target="_blank" rel="noopenner noreferrer">帮助</a> </span><span id="j-jump-sms">短信验证登录</span></div></div><div id="j-sms-box" class="item sms-login-box login-box hide"><div id="j-sms-tip" class="error-tip hide2"><i class="icon error-tip-icon"></i> <span class="txt"></span></div><ul class="form-ul"><li><input type="text" id="dynamicUserName" class="ui-input" autocomplete="off" maxlength="11" placeholder="手机号"> <span class="evet-box clear-btn"><i class="icon clear-btn-icon"></i></span></li><li><input type="text" class="ui-input" id="dynamicPassword" placeholder="短信验证码" maxlength="6"> <input type="button" id="j-msg-verifyCode" class="a-btn pa-right" mobileneedcaptcha="1" value="获取验证码"> <span class="evet-box clear-btn" style="right: 90px;"><i class="icon clear-btn-icon"></i></span></li><li class="last"><input type="button" id="j-dynamicLogin" class="ui-button" value="登录"><div class="jump agreement-box clearfix"><span id="j-agreement-box-sms" class="icon checkbox-icon1"><img class="xy-tip" width="92" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/xy-tip.png" alt=""></span><p class="agreement-txt">未注册手机登录时会自动创建天翼账号，登录即同意<a target="_blank" rel="noopenner noreferrer" href="https://e.dlife.cn/_agreements.html">服务协议</a>和<a target="_blank" rel="noopenner noreferrer" href="https://e.dlife.cn/_agreements.html">隐私政策</a></p></div><div id="j-auto-group-sms" class="free-login hide"><i id="j-auto-login-sms" class="icon checkbox-icon1" data-index="0"></i> <span class="login-select"><em class="login-time-text">7天内自动登录</em> <img class="login-icon-arrow" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-arrow.png" alt=""><div class="login-option-list"><p class="option option-first active" data-time="1">7天内自动登录</p><p class="option" data-time="2">15天内自动登录</p><p class="option option-last" data-time="3">30天内自动登录</p></div></span><img class="login-tips" width="188" style="top: 19px;" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/al-tip2.png" alt=""></div></li></ul><div class="btm-inner clearfix"><span id="j-jump-psw">账号密码登录</span> <a class="feedback fr" href="https://e.dlife.cn/help/registerHelp.do" target="_blank">帮助</a></div></div></div><div id="j-secondValid" class="layer-box hide"><div class="header"><i class="icon back-icon"></i> <span class="txt">安全验证</span></div><div class="login-box verify-box item"><div id="j-safe-tip" class="error-tip hide2"><i class="icon error-tip-icon"></i> <span class="txt"></span></div><div class="content"><ul class="form-ul"><li id="j-second-validate-tit" class="validate-tit">登录验证已通过</li><li id="j-second-validate-text">为保障你的天翼账号安全，当前设备需要进行身份验证。</li><li class="li-phone"> <span id="j-phone" class="phone"></span></li><li class="li-input" id="j-second-sms-valid"><input type="text" id="j-secondValidPhone" class="ui-input" placeholder="短信验证码"> <input type="button" id="j-secondValidCode" class="a-btn pa-right" value="获取验证码"> <span class="evet-box clear-btn" style="right: 90px;"><i class="icon clear-btn-icon"></i></span></li><li id="j-second-psw-valid" class="second-psw-valid"><input type="password" id="j-secondPsw" class="ui-input" placeholder="请输入密码"> <span class="icon eye-icon1 pa-right" data-eye="2"></span> <span class="evet-box clear-btn"><i class="icon clear-btn-icon"></i></span></li><li><input type="button" id="j-secondValidLogin" data-secondvalidtype="sms" class="ui-button" value="确认"></li></ul></div><div class="btm-inner"><a class="feedback fr" href="https://e.dlife.cn/help/registerHelp.do" target="_blank">帮助</a> <a href="https://e.dlife.cn/help/loginHelp.do" target="_blank" rel="noopenner noreferrer">无法进行验证？</a></div></div></div><div id="j-uplink-sms" class="layer-box hide"><div class="header"><i class="icon back-icon J_goBackSMS"></i> <span class="txt">短信上行验证</span></div><div class="login-box up-sms-box"><div id="j-uplink-tip" class="error-tip hide2"><i class="icon error-tip-icon"></i> <span class="txt"></span></div><div class="content pr"><div class="box-inner"><div class="table-wrap"><img width="120" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/up-sms.png" alt=""><table><tbody><tr><td width="80">使用手机号</td><td id="j-tm-mobile">--</td></tr><tr><td>编辑短信</td><td id="j-upLinkSMS-body">--</td></tr><tr><td>发送至</td><td id="j-upLinkSMS-sms">--</td></tr></tbody></table></div><div id="j-uplink-loading" class="box-loading pa"><div class="inner"><img class="mg-rotate" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-loading.png" alt="加载中..."><p class="txt">请稍后</p></div><div id="j-timeout-text" class="div-info pa"><div class="inner"><p class="txt mb10">服务器连接失败</p><input type="button" id="j-upLinkSMS-refresh" class="sm-btn" value="点击刷新"></div></div></div><div class="fail-box pa"><img width="40" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-reload.png" alt=""><div class="txt">短信内容已更换<br>请点重试后重新发送</div></div><div class="suc-box pa"><img src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-success.png" alt=""><div class="txt">验证成功</div></div></div><div><input type="button" id="j-upLinkSMSVerify" class="ui-button ui-button-disable" disabled="disabled" value="确认已发送"></div></div><div class="btm-inner tc">发送短信将由运营商向用户收取普通短信资费</div></div></div><div id="j-ecaptch" class="layer-box hide"><div class="header"><i class="icon back-icon"></i> <span class="txt">安全验证</span></div><div class="ecaptch-box"><div class="__21cn_ecaptch_jigsaw" id="j__21cn_ecaptch"><div class="__21cn_ecaptch_panel"><div class="__21cn_ecaptch_panel-placeholder"><div class="__21cn_ecaptch_bgimg"><img src="" alt="" class="__21cn_ecaptch_bg-img" id="j__21cn_ecaptch_bg"> <img class="__21cn_ecaptch_jigsaw-img" id="j__21cn_ecaptch_jigsaw" src=""> <span class="__21cn_ecaptch_numicon" id="j__21cn_ecaptch_numicon1"></span> <span class="__21cn_ecaptch_numicon" id="j__21cn_ecaptch_numicon2"></span> <span class="__21cn_ecaptch_numicon" id="j__21cn_ecaptch_numicon3"></span></div><div class="__21cn_ecaptch_loadbox" id="j__21cn_ecaptch_loadbox"><div class="__21cn_ecaptch_loadbox__inner"><div class="__21cn_ecaptch_loadicon" id="j__21cn_ecaptch_loadicon"></div><span class="__21cn_ecaptch_loadtext" id="j__21cn_ecaptch_loadtext">加载中</span></div></div><div class="__21cn_ecaptch_top"><div class="__21cn_ecaptch_refresh" id="j__21cn_ecaptch_refresh"></div></div></div></div><div class="__21cn_ecaptch_control" id="j__21cn_ecaptch_control"><div class="__21cn_ecaptch_slide_indicator" id="j__21cn_ecaptch_slide_indicator"></div><div class="__21cn_ecaptch_slider" id="j__21cn_ecaptch_slider"><span class="__21cn_ecaptch_slider__icon"></span></div><div class="__21cn_ecaptch_tips" id="j__21cn_ecaptch_fallbacktip">加载中</div></div></div></div></div><div id="j-login-loading" class="login-loading pa"><div class="div-loading"><p class="img"><img class="mg-rotate" src="https://static.e.189.cn/open/login/page/web/v5.0/static/images/icon-loading.png" alt=""></p><p class="txt">加载中</p></div><div class="div-info pa"><p id="j-config-txt" class="txt">网络开小差了，请刷新重试！</p><input type="button" class="sm-btn" onclick="parent.location.reload();" value="刷新"></div></div></div><script src="https://static.e.189.cn/open/login/page/web/v5.0/static/js/security.js"></script><script defer="defer" src="https://static.e.189.cn/open/login/page/web/v5.0/static/js/captch.min.js?v1.1"></script><script type="text/javascript" src="https://static.e.189.cn/open/login/page/web/v5.0/static/js/App.1684721566703.js"></script></body></html> status: 200 request url: https://open.e.189.cn/api/logbox/separate/web/index.html?appId=cloud&lt=3C1418DDA9D547B46A036B2C021355FB9AB768780806E1761696F0923E5A7D53167A2ED901336C41791115553D5E600953686519CBB9C8E71F2F2D0F5A71B726FE0C259569FE7FF012CC88E54FD6E8581E77F30E&reqId=ac6faba5d40941cbb5885846369fa343 redirect url:

**Comment by kevinzwq** [comment]
为什么alist挂载天翼云盘上传单个文件最大约为10G左右，（已开通会员，正常pc客户端上传文件基本无大小限制），需要怎么设置一下，

**Comment by GHOST1834** [comment]
用天翼云盘客户端挂家庭云一直显示图形验证码错误，不知道怎么解决。

**Comment by wx2020** [comment]
login failed,No toUrl obtained, msg: 您的账号已被冻结，请使用短信或找回密码激活账号
这是咋回事

**Comment by pda8888** [comment]
建议增加天翼企业云

**Comment by piaoying9** [comment]
failed get parent list: failed get dir: object not found  有部分文件夹会提示这个，没办法获取到文件

**Comment by VOfficial404** [comment]


**Reply by youtehub** [reply]
加密吧，别用原图展示，用稳一点的网盘应该可以

**Comment by xianger2017** [comment]
能不能挂分享链接

**Comment by tomseanmy** [comment]
上传报413有大佬知道怎么解决吗

**Comment by W314167** [comment]
天翼云盘客户端出现Get "https://cloud.189.cn/api/portal/unifyLoginForPC.action?appId=8025431004&clientType=10020&returnURL=https%3A%2F%2Fm.cloud.189.cn%2Fzhuanti%2F2020%2FloginErrorPc%2Findex.html&timeStamp=1708079837592": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
怎么解

**Reply by W314167** [reply]
网址是账号登录的界面，怀疑是一直卡在这个界面报超时

**Comment by AlexsNightWithYou** [comment]
使用天翼云客户端，依然有
“Failed init storage: login failed,No toUrl obtained, msg: 图形验证码错误，请重新输入”

**Comment by xiaoxiudog** [comment]
Cannot read properties of undefined (reading 'status')（上传方式秒传）
天翼云盘客户端批量上传文件时部分失败、上传20G大文件时必定失败

**Comment by SSD-3511** [comment]
我在使用天翼云盘客户端登陆时出现：
json: cannot unmarshal number 91960670053560 into Go struct field FamilyInfoResp.familyInfoResp.familyId of type int
这个错误消息表明在尝试初始化存储时，程序发现存储已经创建，但在解析存储配置时出现了问题。具体来说，JSON 数据中的 familyId 字段是一个非常大的数字，但你在 Go 结构体中定义该字段的类型为 int，这导致了解码失败。
